operate/services/protocol.py

```diff
@@ -1397,6 +1397,99 @@ def get_deploy_data_from_safe(
             return [deploy_message]
         return [approve_hash_message, deploy_message]
 
+    def get_safe_b_erc20_transfer_messages(  # pylint: disable=too-many-locals
+        self,
+        safe_a_address: str,
```

Comment:
we don't neet to take a `safe_a_address` here, because this class is already build to support with a "safe a", can be accessed using `self.safe`

Reply:
fixed